### DISCUSSION TITLE
Changes stuck on "Started"

### DISCUSSION THREAD

**Discussion by rhernandezcemssa** [opengisch/QField#3945] [Bugs]
Yesterday I did some fieldwork only to realize when I got back to my desktop that there was an issue with the servers, today according to the status page everything is working again but the edits I did have been stuck since yesterday on "Started" (on the Changes section of the QFieldCloud site) and won't sync, I can't even set them to be re-applied because the actions box is grayed out.

Any solution to this? I really don't want to lose those changes because there were many and it would be very time consuming to do all of them them over again.

**Comment by suricactus** [comment]
@rhernandezcemssa can you please share your username and project name.

**Reply by rhernandezcemssa** [reply]
> @rhernandezcemssa can you please share your username and project name.

username on QFieldCloud is "rhernandez_cemssa" , and the two projects I did edits on yesterday are "cemssa_neteja" and "Cemssa_jardineria2"

**Reply by suricactus** [reply]
Please try again now.

**Reply by rhernandezcemssa** [reply]
Edits have switched back to "Pending" and the action box is selectable again, will check later if they have been successfully applied.

**Reply by rhernandezcemssa** [reply]
> Please try again now.

changes on "Cemssa_jardineria2" applied alright, but the ones on "cemssa_neteja" gave away "Error" on the changes page, trying to apply them again.

**Reply by suricactus** [reply]
The "error" deltas will need standard troubleshooting and it is not part of the reported problem. Check job logs for more info why you get the error and if still required, open a new issue.

**Comment by uvesco** [comment]
It seems I have the same issue with those changes: stuck on started status and no way to reapply those changes. Is there another way to manage it different to copy it by hand in new db entries?
thank you very much

**Reply by suricactus** [reply]
This will be solved by https://github.com/opengisch/qfieldcloud/pull/941

If you are a user of app.qfield.cloud, please contact support.qfield.cloud to fix your issue.
